lib/Arch/Sleigh/Arch.cpp

```diff
@@ -201,8 +202,8 @@ bool SleighDecoder::DecodeInstruction(uint64_t address,
         << "Should always emit branch taken var for conditional instruction";
   }
 
-  inst.SetLifter(std::make_shared<SleighLifterWithState>(res_cat->second,
-                                                         this->GetLifter()));
+  inst.SetLifter(std::make_shared<SleighLifterWithState>(
+      res_cat->second, std::move(context_values), this->GetLifter()));
```

Comment:
Call the `context.GetContextValues()` here? Or is there a reason copy must happen earlier? If it is so, add comment.